frontend/src/Editor/Editor.jsx

```diff
@@ -161,6 +166,17 @@ class Editor extends React.Component {
     });
   };
 
+  // 1. When we receive an undoable action – we can always undo but cannot redo anymore.
+  // 2. Whenever you perform an undo – you can always redo and keep doing undo as long as we have a patch for it.
+  // 3. Whenever you redo – you can always undo and keep doing redo as long as we have a patch for it.
+  initComponentVersioning = () => {
+    this.currentVersion = -1;
```

Comment:
nitpick: will this be confusing later b/w app's current version and immer patch's current version? ́😅

Reply:
Actually app version can be version 1, but the immer patch's current version can be anywhere between a range of 1-100, 

But thanks for bringing this up, one issue I found is this doesn't reset on changing the `app` version, I feel on change of app version calling `initComponentVersioning` should suffice

Reply:
I meant we store `currentVersionId` for apps at db, was speaking in terms of that.